docs/built-in-functions.rst

```diff
@@ -196,112 +329,259 @@ contracts.
+    .. code-block:: python
 
-.. py:function:: raw_log(topics: bytes32[4], data: Bytes) -> None
+        >>> ExampleContract.foo(2, 3)
+        8
+        >>> ExampleContract.foo(100, 100)
+        59041770658110225754900818312084884949620587934026984283048776718299468660736
 
-    Provides low level access to the ``LOG`` opcodes, emitting a log without having to specify an ABI type.
+.. py:function:: sqrt(d: decimal) -> decimal
 
-    * ``topics``: List of ``bytes32`` log topics
-    * ``data``: Unindexed event data to include in the log, bytes or bytes32
+    Return the square root of the provided decimal number, using the Babylonian square root algorithm.
 
-    This method provides low-level access to the ``LOG`` opcodes (``0xA0``..``0xA4``). The length of ``topics`` determines which opcode will be used. ``topics`` is a list of bytes32 topics that will be indexed. The remaining unindexed parameters can be placed in the ``data`` parameter.
+    .. code-block:: python
 
+        @external
+        @view
+        def foo(d: decimal) -> decimal:
+            return sqrt(d)
 
-.. py:function:: create_forwarder_to(target: address, value: uint256 = 0) -> address
+    .. code-block:: python
 
-    Duplicates a contract's code and deploys it as a new instance, by means of a ``DELEGATECALL``.
+        >>> ExampleContract.foo(9.0)
+        3.0
 
-    * ``target``: Address of the contract to duplicate
-    * ``value``: The wei value to send to the new contract address (Optional, default 0)
+.. py:function:: uint256_addmod(a: uint256, b: uint256, c: uint256) -> uint256
 
-    Returns the address of the duplicated contract.
+    Return the modulo of ``(a + b) % c``. Reverts if ``c == 0``.
+
+    .. code-block:: python
+
+        @external
+        @view
+        def foo(a: uint256, b: uint256, c: uint256) -> uint256:
+            return uint256_addmod(a, b, c)
+
+    .. code-block:: python
+
+        >>> (6 + 13) % 8
+        3
+        >>> ExampleContract.foo(6, 13, 8)
+        3
+
+.. py:function:: uint256_mulmod(a: uint256, b: uint256, c: uint256) -> uint256
+
+    Return the modulo from ``(a * b) % c``. Reverts if ``c == 0``.
+
+    .. code-block:: python
+
+        @external
+        @view
+        def foo(a: uint256, b: uint256, c: uint256) -> uint256:
+            return uint256_mulmod(a, b, c)
+
+    .. code-block:: python
+
+        >>> (11 * 2) % 5
+        2
+        >>> ExampleContract.foo(11, 2, 5)
+        2
+
+Utilities
+=========
+
+.. py:function:: as_wei_value(_value, unit: str) -> uint256
+
+    Take an amount of ether currency specified by a number and a unit and return the integer quantity of wei equivalent to that amount.
+
+    * ``_value``: Value for the ether unit. Any numeric type may be used, however the value cannot be negative.
+    * ``unit``: Ether unit name (e.g. ``"wei"``, ``"ether"``, ``"gwei"``, etc.) indicating the denomination of ``_value``. Must be given as a literal string.
```

Comment:
```suggestion
    * ``unit``: Ether unit name (e.g. ``"wei"``, ``"ether"``, ``"gwei"``, etc.) indicating the denomination of ``_value``. Must be given as decimal literal.
```

Reply:
It's a literal string. Maybe you mean `_value` ?

Reply:
yes, sorry